typescript/big-practice/src/scripts/views/listNoteView.ts

```diff
@@ -654,21 +663,27 @@ export default class ListNoteView {
    * @param {Object} noteElement is note element
    * @param {function} deleteNote is function delete transmitted from from the model
    */
-  bindDeleteNote(noteElement, deleteNote) {
+  bindDeleteNote(noteElement: HTMLElement, deleteNote: (id: string) => void) {
     const note = selectDOMById(`${noteElement.id}`);
     const headerAfterSelect = selectDOMClass('.header-after-select');
-    const iconDeleteElement = note.querySelectorAll('.note-btn .icon-delete');
-    const handler = (e) => {
+
+    const handler = (e: Event) => {
       e.stopPropagation();
-      const noteId = e.target.getAttribute('data-id');
+      const noteId = (e.target as HTMLElement).getAttribute('data-id');
 
-      deleteNote(noteId);
-      this.elementHelpers.removeSelected();
-      this.elementHelpers.translateYElement(headerAfterSelect, '-200');
+      if (noteId && headerAfterSelect) {
+        deleteNote(noteId);
+        this.elementHelpers.removeSelected();
+        this.elementHelpers.translateYElement(headerAfterSelect, '-200');
```

Comment:
If this handle css of the UI, pls use class toggle to custom it instead of write directly style code in script